gallery/src/components/NxThreatIndicatorLegend/NxThreatIndicatorLegendPage.tsx

```diff
@@ -0,0 +1,113 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+import { NxCode, NxInfoAlert, NxP, NxTable } from '@sonatype/react-shared-components';
+
+import { GalleryDescriptionTile, GalleryExampleTile } from '../../gallery-components/GalleryTiles';
+
+import NxThreatIndicatorLegendHorizontalExample from './NxThreatIndicatorLegendHorizontalExample';
+import NxThreatIndicatorLegendVerticalExample from './NxThreatIndicatorLegendVerticalExample';
+
+const NxThreatIndicatorLegendHorizontalExampleCode = require('./NxThreatIndicatorLegendHorizontalExample?raw'),
+    NxThreatIndicatorLegendVerticalExampleCode = require('./NxThreatIndicatorLegendVerticalExample?raw');
+
+const NxThreatIndicatorLegendPage = () =>
+  <>
+    <GalleryDescriptionTile>
+      <NxP>
+        <NxCode>NxThreatIndicatorLegend</NxCode> is an element to display the legend for
+        various threat levels. The legend can be in two formats: horizontal and vertical.
+      </NxP>
+      <NxInfoAlert>
+        By default, <NxCode>NxThreatIndicatorLegend</NxCode> renders a horizontal legend.
+        To render a vertical legend, the prop <NxCode>vertical</NxCode> needs to be provided.
+      </NxInfoAlert>
+      <NxP>
+        <NxCode>NxThreatIndicatorLegend</NxCode> supports displaying the legend for one of
```

Comment:
More than one of, really

Reply:
> More than one of, really

Fixed. 030aa9e072ee9e74357a3508bfcadc22a079f13c